code/style.css

```diff
@@ -1,7 +1,333 @@
+
+form {
+  max-width: 50vw;
+  margin-left: 15px;
+}
+
+legend {
+  font-family: "Lexend Giga", sans-serif;
+  color: grey;
+  text-decoration: none;
+  font-size: 20px;
+  border-bottom: 0px !important;
+  margin-left: 15px;
+}
+
+.form-group {
+  color: white;
+  font-family: "Lexend Giga", sans-serif;
+  font-weight: 200;
+  font-size: 20px;
+  border-radius: 5px;
+  line-height: 30px;
+  background-color: transparent;
+  border: 2px solid #fdffb6;
+  transition: all 0.5s;
+  padding: 15px;
+  margin-bottom: 15px;
+  width: 50vw;
+  box-sizing: border-box;
+  outline: 0;
+  color: grey;
+}
+
+.form-group:focus {
+  border: 2px solid #fcff99;
+}
+
+.selection {
+  background-color: lightgrey;
+  color: grey;
+  font-family: "Lexend Giga", sans-serif;
+}
+select {
+  background: lightgrey;
+}
+
+.submit {
+  font-family: "Crimson", Arial, Helvetica, sans-serif;
+  width: 50vw;
+  background: #fcff99;
+  border-radius: 5px;
+  border: 0;
+  cursor: pointer;
+  color: #908aa8;
+  font-size: 24px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  transition: all 0.3s;
+  margin-top: 5px;
+  font-weight: 700;
+}
+.submit:hover {
+  background: #ececd3;
+}
+
+.input_data label {
+  margin-bottom: 5px;
+  padding-bottom: 1em;
+  color: grey;
+  font-family: "Lexend Giga", sans-serif;
+}
+
+.radio-buttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: grey;
+  background-color: lightgrey;
+  font-family: "Lexend Giga", sans-serif;
+}
+
+.checkbox,
+checkbox {
+  margin-left: 25px;
+  color: grey;
+  background-color: lightgrey;
+  font-family: "Lexend Giga", sans-serif;
+}
+/***accordion***/
+
+.accordion-intro {
+  margin-bottom: 40px;
+  color: grey;
+  font-size: 20px;
+  font-family: "Lexend Giga", sans-serif;
+}
+.accordion-container {
+  border: 2px solid #fdffb6;
+  border-radius: 5px;
+  margin-bottom: 15px;
+  margin-top: 15px;
+  color: grey;
+  font-family: "Lexend Giga", sans-serif;
+  font-size: 20px;
+  padding: 4px;
+  margin: 15px;
+}
+
+.accordion-header {
+  margin: 0;
+  padding: 15px;
+  cursor: pointer;
+  font-size: 20px;
+}
+
+.accordion-text {
+  border-top: 2px solid #999;
+  margin: 0;
+  padding: 15px;
+  font-size: 18px;
+}
+
+.accordion-text-container {
+  height: 0;
+  overflow: scroll;
+  transition: height 0.3s ease;
+}
+
+.accordion-text-container.active {
+  height: 550px;
+}
+
+/***main section (signup form and accordion for tablet/mobile***/
+
+@media screen and (max-width: 1020px) {
+  .main {
+    flex-direction: column;
+  }
+  .signup {
+    margin-left: -15px;
+  }
+  .form {
+    width: 100vw;
+  }
+  .form-group {
+    width: 100vw;
+  }
+  .submit {
+    width: 100vw;
+  }
+}
+
+/***footer***/
+
+.footer {
+  text-align: center;
+  display: inline-block !important;
+  background-color: lightgrey;
+  width: 100%;
+  border: 2px solid #fdffb6;
+  margin-top: 50px;
+  position: relative;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 70px;
+  line-height: 50px;
+}
```

Comment:
You site got a tiny bit of side scroll, which is okey, but I think it's good practise to check what's causing this 👋